mtp_noms_ops/apps/security/utils.py

```diff
@@ -38,7 +38,6 @@ def convert_date_fields(object_list, include_nested=False):
     supported yet.
     """
     fields = ('started_at', 'received_at', 'credited_at', 'refunded_at', 'created', 'triggered_at', 'actioned_at')
-    parsers = (parse_datetime, parse_date)
```

Comment:
this change makes me quite uneasy. a date string would have a random time part invented and have the wrong type (even if it has a superset of attrs).

a smaller and less scary change would be to look for `'T'` in the string and then use `parse_datetime`, `parse_date ` otherwise